### ISSUE TITLE
HikariConfig Warning

### ISSUE THREAD

**Issue by janlisse** [playframework/play-slick#264]
When using latest play 2.4.0-RC5 with play-slick-1.0.0-RC3 with the following application.conf:

```
slick.dbs {
  default {
    driver = "slick.driver.MySQLDriver$"
    db.driver = "com.mysql.jdbc.Driver"
    db.url = "jdbc:mysql://mysql:3306/mmadb?user=mma&password=secret"
  }
}
````

On initialization of the Hikari connection pool i get the following warning:

[warn] c.z.h.HikariConfig - validationTimeout is greater than connectionTimeout, setting validationTimeout to connectionTimeout.

How can i fix this warning?
According to the [Play documentation](https://www.playframework.com/documentation/2.4.x/SettingsJDBC) default validationTimeout ought to be 5 seconds whilst connectionTimout ought to be 30 seconds. Seems like this is not the case.

**Comment by evandu** [comment]
me too

**Comment by gguan** [comment]
+1

**Comment by dotta** [comment]
Please, before opening a ticket discuss the problem in the [playframework mailing list](https://groups.google.com/forum/#!forum/play-framework). If an issue is confirmed, then open a ticket.

The HikariCP pool is configured (and managed) by Slick, and it does not use the Play defaults. Hence, I would suggest to ask the question in the [Slick mailing list](https://groups.google.com/forum/#!forum/scalaquery).

**Comment by dotta** [comment]
And, by the way, here is a link to the fix in Slick https://github.com/slick/slick/pull/1175

**Comment by dotta** [comment]
The fix will be available with Slick 3.0.1 (and we will release a play-slick 1.0.1 then).
